uint8 id1
uint8 id2
uint8 id3
int32 value1
int32 value2
int32 value3
